sdk/gperftools/gperftools-2.5/src/base/vdso_support.cc

```diff
@@ -43,7 +43,6 @@
 #include <stddef.h>   // for ptrdiff_t
 
 #include "base/atomicops.h"  // for MemoryBarrier
-#include "base/linux_syscall_support.h"
```

Comment:
Does this change come from upstream?

Reply:
It does not. I mistakenly thought all third-party dependencies were in the external/ directory.

Reply:
Oh, but I see they had the same idea https://github.com/gperftools/gperftools/commit/977e0d45003514d088e4bb83474c985094acee47#diff-2599408afdb3c2829102d83ae47df1cb

Reply:
Is this patch accepted by upstream, if so we can merge it.